CHANGES.md

```diff
@@ -57,3 +57,5 @@ Format of the entries must be.
 * Updated Metronome to 0.5.0. (DCOS_OSS-2338)
 
 * Updated OTP version to 20.3.2 (DCOS_OSS-2378)
+
+* Updated dcos-checks timeouts
```

Comment:
This should be more descriptive, and it needs a ticket. Instructions for adding a changelog entry are at the top of this file.

Reply:
Is this in the correct section? Your 1.11 PR has it under "Fixed and improved": https://github.com/dcos/dcos/pull/2977